db/raw/add-hotel-plugin.sql

```diff
@@ -0,0 +1,1576 @@
+--
+-- PostgreSQL database dump
+--
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+--
+-- Name: sqitch; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA sqitch;
```

Comment:
No, migrations shouldn't be sequential. The purpose of these migration files is to allow any event to tack on a new plugin and migrate its database - having to recreate a set of migrations for each event would be an awful lot of work.

As I noted earlier though, the process by which we create migrations could probably use some fine-tuning. :)